### PR TITLE
Issues/48 - Redshift distributions from file

### PR DESCRIPTION
**Pull request by fjaviersanchez**
Trying to address #48. This PR adds the ability to read either a text file (with format z, dndz_0, dndz_1, dndz_2, ...) or an npy file (with the same format as the one that is returned by CCLX) and assign it to a tracer.

### CODE REVIEW

File: augur/tracers/two_point.py [hunk 1]
```diff
@@ -221,3 +221,34 @@ def to_dict(self):
...
+                self.Nz = data[()]['bins'][ibin]
+        elif 'ascii' in format:
+            data = np.loadtxt(input_file)
+            self.z = data[:, 0]
+            if ibin is None:
+                self.Nz = data[:, 1]
+            else:
+                self.Nz = data[:, ibin+1]
+        else:
+            raise NotImplementedError('Only ascii files or npy files are currently supported')
```
Comment (arthurmloureiro) [inline augur/tracers/two_point.py:252]: Shall we add a check to see if the number of dndz columns in the file matches the `nbins` keyword from the configuration files?

**Review by arthurmloureiro — CHANGES_REQUESTED** [review]
Awesome! I tested here and it works out of the box! I'd just add a small check to see if the number of bins in the config file matches the number of bins in the file.

File: augur/tracers/two_point.py (hunk 1 above)
Comment (arthurmloureiro) [reply augur/tracers/two_point.py:252]: I realised this is done by construction in the `generate.py` structure!

**Review by arthurmloureiro — APPROVED** [review]

**Comment by fjaviersanchez** [comment]
yes, that's a good point. I should add some safeguards :)

**Pull request merged**